tests/components/autofill.test.ts

```diff
@@ -101,13 +101,7 @@ describe("Autofill component", () => {
     const sheetId = model.getters.getActiveSheetId();
     const x = HEADER_WIDTH + model.getters.getColDimensions(sheetId, 1)!.end + 20;
     const y = model.getters.getRowDimensions(sheetId, 0)!.start + 20;
-    fixture.querySelector(".o-grid")!.dispatchEvent(
-      new WheelEvent("wheel", {
-        clientX: x,
-        clientY: y,
-        bubbles: true,
-      })
-    );
+    triggerWheelEvent(".o-grid", { clientX: x, clientY: y });
```

Comment:
This test is kinda sus'. `clientX/Y` for wheel events ? The test don't work when replacing them with `deltaX/Y`

Reply:
it's a bit shitty but occurs because we handle the wheel event with the same callback as 'mouseMove' in the drag&drop helpers, mousemove, which ultimately works with clientX/Y. It would be more correct to force a clientX,Y  at start (here 40,40) then trigger the wheel event with a delta and ensure that the position tooltip position has not moved (from a client pov) but that the dispatched command takes the scrolling into account properly.